util/personal_names.py

```diff
@@ -149,24 +149,34 @@ def display_name_to_sort_name(display_name):
     # name has title, first, middle, last, suffix, nickname
     name = HumanName(display_name)
 
-    if name.nickname:
-        name.nickname = '(' + name.nickname + ')'
 
     # Note: When the first and middle names are initials that have come in with a space between them,
     # let them keep that space, to be consistent with initials with no periods, which would be more
     # easily algorithm-recognized if they were placed separately. So:
     # 'Classy, A. B.' and 'Classy Abe B.' and 'Classy A. Barney' and 'Classy, Abe Barney' and 'Classy, A B'.
+
+    # This might go after a comma, or it might be someone's entire
+    # name.
+    base_name = u' '.join([name.title, name.first, name.middle, name.suffix]).strip()
     if not name.last:
         # Examples: 'Pope Francis', 'Prince'.
-        sort_name = u' '.join([name.first, name.middle, name.suffix, name.nickname])
-        if name.title:
-            sort_name = u''.join([name.title, ", ", sort_name])
+        sort_name = base_name
     else:
-        sort_name = u' '.join([name.first, name.middle, name.suffix, name.nickname, name.title])
```

Comment:
So now the title always goes first in the base name and not last?

Reply:
That's right. I made that decision based on https://nameparser.readthedocs.io/en/latest/, which lists three different ways of writing a name with Title always coming before the given name.

When I think of "title" I think of stuff like "5th Earl of Combsbury" which goes last, but by "title" nameparser means stuff like "Pope" or "Dr."